js/app/app.js

```diff
@@ -17,5 +17,60 @@ document.addEventListener("DOMContentLoaded", () => {
 })
 
 
+//** contributors section */
+
+/** ### async func get list contributors for repo from github */
+const getContributors = async (repoName) => {  
+    let request = await fetch(`https://api.github.com/repos/${repoName}/contributors`, {
```

Comment:
Wrap the `await` sections in a `try/catch` to handle any errors.

Reply:
Done